static/app/styles/prism.tsx

```diff
@@ -127,6 +127,9 @@ export const prismStyles = (theme: Theme) => css`
 
   pre[data-line] {
     position: relative;
+    code {
+      background: unset;
```

Comment:
Without this reset, each token would inherit the background which would not work well with the line highlight

Reply:
I think we can make this more general move it inside the `pre[class*='language-']` block above
```css
pre[class*='language-'] {
  …
  code {
      background: unset;
  }
}
```

`code` spans inside `pre` blocks don't need backgrounds, since `pre` blocks already have backgrounds.